packages/web/src/components/organisms/TransactionForm/index.tsx

```diff
@@ -101,7 +130,8 @@ export const TransactionForm = ({ propertyAddress }: Props) => {
       <CancelStakingCard
         onClickCancel={handleCancelStaking}
         onClickWithdraw={handleWithdrawStaking}
-        disabledWithdraw={!isCancelCompleted}
+        remainBlocks={remainBlocks}
+        isCompleted={withdrawable}
```

Comment:
`disabledWithdraw` has been removed because it can be covered by the state of `isCompleted` .